AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -5211,6 +5211,7 @@ versteigerungspool.de#%#//scriptlet('set-cookie', 'consent', 'true')
 wolt.com#%#//scriptlet('set-cookie', 'CONSENT', '1')
 gittigidiyor.com#%#//scriptlet('set-cookie', 'ggPolicyBoxIsHidden', 'true')
 ballhandel.de#%#//scriptlet('set-cookie', 'allowCookie', '1')
+bold.dk#%#AG_onLoad(function() {if (!document.cookie.includes("euconsent-v2")) {var g = new MutationObserver(function() {var b = document.querySelector('button[aria-label="Accepter"]');b&&b.click();});g.observe(document, {childList: !0,subtree: !0});setTimeout(function() {g.disconnect()}, 1E4)}});
```

Comment:
It seems that rule doesn't work on my end. @Alex-302 @stanislav-atr please take a look.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/109145912-32600f80-7774-11eb-9d07-a6963508b19c.png)

</details><br/>

Reply:
This was due to the fact that the cookies was not open in the window. Exchanged the rule.